src/patternfly/demos/Alert/alert-template-horizontal-form.hbs

```diff
@@ -28,7 +28,7 @@
       {{/form-label}}
     {{/form-group-label}}
     {{#> form-group-control}}
-      {{#> form-control controlType="input" input=true form-control--IsError="true" form-control--IsRequired="true" form-control--attribute=(concat 'type="text" value="patternfly@patternfly.com" id="' form-group--id '" name="' form-group--id )}}{{/form-control}}
+      {{#> form-control controlType="input" input='true' form-control--IsRequired='true' form-control--attribute=(concat 'type="text" value="patternfly@patternfly.com" id="' form-group--id '" name="' form-group--id '"')}}{{/form-control}}
```

Comment:
Just a comment, I see you added single quotes around an unquoted `true` and changed double quotes to single quotes. For these boolean things, they don't need quotes at all, and typically we use double quotes to wrap param values except values that will contain double quotes (most commonly like `--attribute='something="whatever"'`)